features/click.yml

```diff
@@ -0,0 +1,5 @@
+name: Click
+description: The `click` method simulates a mouse click on an element. When called on an element, the element's click event is fired.
+spec: https://html.spec.whatwg.org/multipage/interaction.html#dom-click-dev
+compat_features:
+  - api.HTMLElement.click
```

Comment:
Honestly, I think this is not far off from calling `MouseEvent("click", …)`. I think this key belongs in `mouse-events` and we can drop this feature.